components/atoms/Interests/index.js

```diff
@@ -7,7 +7,7 @@ export default function InterestSelect({ interest, setInterest }) {
       <select
         value={interest}
         onChange={(e) => { setInterest(e.target.value); }}
-        placeHolder="Select Interest"
+        placeholder="Select Interest"
```

Comment:
seems like we have implemented same fix here. haha

Reply:
lol. No wahala